snippets/customers/login-form.liquid

```diff
@@ -2,15 +2,19 @@
 {% include 'form-errors-custom' %}
 
 <fieldset ng-keypress="login($event)">
-    <div class="form-group">
+    <div class="form-group" ng-controller="flyingController">
```

Comment:
ng-controller="flyingController" also need to be deleted

Reply:
deleted